tests/e2e/create/create_test.go

```diff
@@ -299,22 +314,8 @@ var _ = Describe("Test 'create' command", Ordered, func() {
 			template, err := readModuleTemplate(templateOutputPath)
 			Expect(err).ToNot(HaveOccurred())
 			descriptor := getDescriptor(template)
-			Expect(descriptor).ToNot(BeNil())
-			Expect(descriptor.SchemaVersion()).To(Equal(v2.SchemaVersion))
-			Expect(template.Name).To(Equal("template-operator-1.0.0"))
 
-			By("And spec.info should be correct")
-			Expect(template.Spec.ModuleName).To(Equal("template-operator"))
-			Expect(template.Spec.Version).To(Equal("1.0.0"))
-			Expect(template.Spec.Info.Repository).To(Equal("https://github.com/kyma-project/template-operator"))
-			Expect(template.Spec.Info.Documentation).To(Equal("https://github.com/kyma-project/template-operator/blob/main/README.md"))
-			Expect(template.Spec.Info.Icons).To(HaveLen(1))
-			Expect(template.Spec.Info.Icons[0].Name).To(Equal("module-icon"))
-			Expect(template.Spec.Info.Icons[0].Link).To(Equal("https://github.com/kyma-project/template-operator/blob/main/docs/assets/logo.png"))
-
-			By("And annotations should be correct")
-			annotations := template.Annotations
-			Expect(annotations[shared.IsClusterScopedAnnotation]).To(Equal("false"))
+			validateMinimalModuleTemplate(template, descriptor)
```

Comment:
The shared part of the MT writing between normal and dry-run has been moved to this helper function